File: App/src/main/java/it/gruppoinfor/home2work/utils/RouteUtils.java
package it.gruppoinfor.home2work.utils;

import com.google.android.gms.maps.model.LatLng;
import com.google.android.gms.maps.model.LatLngBounds;

import java.util.List;


public class RouteUtils {

    public static LatLngBounds getRouteBounds(List<LatLng> locations) {

        LatLngBounds.Builder builder = new LatLngBounds.Builder();

        for (LatLng location : locations) {
            builder.include(location);
        }

        return builder.build();
    }
}

File: App/src/main/java/it/gruppoinfor/home2work/utils/RouteUtils.kt
package it.gruppoinfor.home2work.utils

import com.google.android.gms.maps.model.LatLng
import com.google.android.gms.maps.model.LatLngBounds


object RouteUtils {

    fun getRouteBounds(locations: List<LatLng>): LatLngBounds {

        val builder = LatLngBounds.Builder()

        for (location in locations) {
            builder.include(location)
        }

        return builder.build()
    }
}
